netbox/core/graphql/types.py

```diff
@@ -1,21 +1,34 @@
-from core import filtersets, models
+from typing import Annotated, List
+
+import strawberry
+import strawberry_django
+
+from core import models
 from netbox.graphql.types import BaseObjectType, NetBoxObjectType
+from .filters import *
 
 __all__ = (
     'DataFileType',
     'DataSourceType',
 )
 
 
+@strawberry_django.type(
+    models.DataFile,
+    exclude=('data',),
```

Comment:
`strawberry_django` seems to demand a list here.
```suggestion
    exclude=['data'],
```